packages/nimble-components/src/table/tests/table-matrix.stories.ts

```diff
@@ -37,46 +37,44 @@ const data = [
         firstName: 'Milhouse',
         lastName: 'Van Houten',
         age: 10,
-        favoriteColor: 'Crimson',
         quote: "Not only am I not learning, I'm forgetting stuff I used to know!"
     },
     {
         id: '2',
         firstName: null,
         lastName: null,
-        favoriteColor: null,
         quote: null
     },
     {
+        id: '3',
         firstName: 'Jacqueline',
         lastName: 'Bouvier',
         age: 80,
         quote: "I have laryngitis. It hurts to talk, so I'll just say one thing. You never do anything right.",
-        id: '3',
         parentId: undefined
     },
     {
+        id: '4',
         firstName: 'Selma',
         lastName: 'Bouvier',
         age: 45,
         quote: "Hey relax. I've told ya' I've got money. I bought stock in a mace company just before society crumbled.",
-        id: '4',
         parentId: '3'
     },
     {
+        id: '',
         firstName: 'Marge',
         lastName: 'Simpson',
         age: 35,
         quote: "Oh, I've Always Wanted To Use Rosemary In Something!",
-        id: '',
         parentId: '3'
     },
     {
+        id: '7',
```

Comment:
nit: Is there a reason to jump from an id of 4 to an id of 7 here? I don't think it's necessarily a problem, but if there isn't a reason to, this fake data will probably be more maintainable if we use consecutive id values.